### ISSUE TITLE
Keras support in Mantisshrimp

### ISSUE THREAD

**Issue by ai-fast-track** [airctic/icevision#260]
## 🚀 Feature
It would good that Mantisshrimp supports TensorFlow (TF) through the Keras library. This will position us a Complete Agnostic Object Detection Framework, and we will be walking in HuggingFace footsteps.

**Solution**
We can create a minimal API that serves us a proof of concept that we can support TF through Keras. Once the proof the concept is finished, we can iterate and improve the new API. 

After announcing Mantis at the Incubator, we can start reaching out to both Keras and AutoKeras communities to ask for their help.

**Resources**
[Keras Documentation](https://keras.io/)

[AutoKeras Documentation](https://autokeras.com/)

**Comment by FraPochetti** [comment]
TF support is out of scope
